src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
+    aria-expanded="false"
+    aria-labelledby="downshift-0-label"
+    autocomplete="off"
+    data-testid="input"
+    id="downshift-0-input"
+    role="listbox"
+    value="item"
+  />
+  <button
+    aria-expanded="false"
+    aria-haspopup="true"
+    aria-label="open menu"
+    data-testid="button"
+    data-toggle="true"
+    role="button"
+    type="button"
+  />
+  <ul
+    aria-labelledby="downshift-0-label"
+    data-testid="menu"
+    id="downshift-0-menu"
+    role="listbox"
+  >
+    <li
+      aria-selected="true"
```

Comment:
I see this is set to `true` because it is the option that matches the set value of the `input`, just a note to make sure it's set to true also when it is "in focus" or the active options I've arrowed to and is the current value of `aria-activedescendant` on the `input.

Reply:
It depends I think. The default use-case for downshift is that the selected item has to be explicitly selected. Then there's a concept of a highlighted item which can only be selected when clicked or on enter-keypress. This behavior can be changed to select the highlighted item, and in that case this will behave properly.

Reply:
Sure, I guess i’m Aware of an old voiceover quirk. If you don’t set `aria-selected=“true”` when `aria-activedescendant` matches an option ID, it often won’t announce the option.